File: test/fixtures/go-project/internal/application/order_service.go
package application

import "github.com/example/test-project/internal/domain"

// OrderService orchestrates order operations
type OrderService struct{}

// CreateOrder creates a new order
func (s *OrderService) CreateOrder(items []string) *domain.Order {
	order := domain.Order{Items: items}
	if !domain.ValidateOrder(order) {
		return nil
	}
	return &order
}

File: test/fixtures/ts-project/src/application/order.service.ts
import type { Order } from '@domain/order';
import { validateOrder } from '@domain/order';

// OrderService orchestrates order operations
export class OrderService {
  createOrder(items: string[]): Order | null {
    const order: Order = { id: crypto.randomUUID(), items };
    if (!validateOrder(order)) {
      return null;
    }
    return order;
  }
}
